### ISSUE TITLE
Channel image aspect ratio rendered to 2:1 instead of 1:1 on squared images within channel button

### ISSUE THREAD

**Issue by giacomouberti** [globaleaks/globaleaks-whistleblowing-software#4303]
### What version of GlobaLeaks are you using?

5.0.20

### What browser(s) are you seeing the problem on?

Chrome

### What operating system(s) are you seeing the problem on?

macOS

### Describe the issue

After the update to V5 all channel picture that were previously uploaded as a squared image are now displayed with a forced 2:1 aspect ratio resulting in a stretched picture to fill the button with the channel name.

On the channel page before the questionnaire the image is displayed correctly

**Comment by giacomouberti** [comment]
Sure @evilaliv3, here's a screenshot from a demo site we're using
<img width="1536" alt="Screenshot 2024-11-11 alle 14 32 16" src="https://github.com/user-attachments/assets/1a8f49f3-4fb8-434c-b130-cd22e7822fc3">

and here's the html rendered code of one of the card
```
<div class="select-card">
        <div class="row">
          <div class="col-md-12 title">SAFEGUARDING</div>
        </div>
        
          <div class="row ng-star-inserted">
            <div class="col-md-4">
              
                <img alt="context picture" width="500" height="500" class="context-image ng-star-inserted" loading="lazy" fetchpriority="auto" ng-img="true" src="/s/09d47ac0-6415-40d5-9293-93c137123f45">
              <!----></div>
          </div>
        <!----><!----></div>
```

Here's a link to the image https://demo.wb.teseoerm.com/s/09d47ac0-6415-40d5-9293-93c137123f45

**Comment by evilaliv3** [comment]
Thank you @giacomouberti 

It is effectively a regression on release 4.

We are proceeding issuing a patch that will be included in the minor release 5.0.25 that we we will be issuing today.

**Comment by evilaliv3** [comment]
Thank you @giacomouberti 

would you please retry with release 5.0.25?

**Comment by giacomouberti** [comment]
Hi there @evilaliv3, I just installed the updated release but now the image appears like a missing source and indeed in the rendered html this is the output missing, in fact, the source
```
<div class="select-card">
        <div class="row">
          <div class="col-md-12 title">DEMO WHISTLEBLOWING</div>
        </div>
        
          <div class="row ng-star-inserted">
            <div class="col-md-4">
              
                <img alt="context picture" class="context-image ng-star-inserted">
              <!----></div>
          </div>
        <!----><!----></div>
```
In the console on the client side you get the error `"TypeError: Cannot read properties of undefined (reading 'toString')"`

The image isn't visible even from the backend in the context settings
<img width="1198" alt="Screenshot 2024-11-11 alle 23 02 42" src="https://github.com/user-attachments/assets/b2b45b8f-86c7-4005-90a3-a071716b716f">
<img width="1434" alt="Screenshot 2024-11-11 alle 23 02 53" src="https://github.com/user-attachments/assets/1eb9ab39-7577-4d4c-b8cb-bf6c43251a34">

When you choose one of the context on the questionnaire's page the picture is displayed regularly
<img width="1436" alt="Screenshot 2024-11-11 alle 23 22 20" src="https://github.com/user-attachments/assets/7bf6f2d3-e7db-4335-b1db-0b6b4d80a71a">

**Comment by evilaliv3** [comment]
@giacomouberti : The issue on the loading of images was a different issue appeared in 5.0.25.

Now both the two issues should be fixed in release 5.0.26
